api/data_refinery_api/utils.py

```diff
@@ -0,0 +1,24 @@
+def check_filters(view, special_filters=None):
```

Comment:
This function is defined both here and in common. I think this is the more appropriate place though since I don't think it'd be used outside the API.